File: 0009-palindrome-number/0009-palindrome-number.c
bool isPalindrome(int x) {
   if(x<0|| x!=0 && x%10==0) return false;
   int reverse=0;
   while(x>reverse){
      reverse=reverse*10 + x%10;
      x/=10;
   }
   return (reverse==x||x==reverse/10);
}

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int temp = x;
        int rev = 0;
        while(x > 0)
        {
             rev = rev*10 + x%10;
             x /= 10; 
        }
        System.out.print(rev);
        if(temp==rev) return true;
        else return false;

    }
}